conf/modules.config

```
@@ -158,6 +158,11 @@ params {
         'bin_summary' {
             publish_dir     = "GenomeBinning"
         }
+        prokka {
+            args            = ""
```

```suggestion
            args            = "--metagenome"
```

>   --metagenome      Improve gene predictions for highly fragmented genomes (default OFF)

and we do expect here (mostly) fragmented genomes.